events/src/main/java/org/opentest4j/reporting/events/root/Events.java

```diff
@@ -58,4 +59,17 @@ public static DocumentWriter<Events> createDocumentWriter(NamespaceRegistry name
 		return DocumentWriter.create(ELEMENT, namespaceRegistry, xmlFile);
 	}
 
+	/**
+	 * Create a new {@link DocumentWriter} for the event-based reporting format.
+	 *
+	 * @param namespaceRegistry the namespace registry to use
+	 * @param writer            the writer to write to
```

Comment:
We should document that the supplied `Writer` will be closed when closing the returned `DocumentWriter`.